aria_data_loaders/aria_data_utils/aria_live_streamer.py

```diff
@@ -0,0 +1,469 @@
+import os
```

Comment:
file-level docstring to explain all the frames we work with in ROS and assume available in this file.

Reply:
Should that be somewhere in the readme?

Reply:
moved to README.md and linked to both `episodic_memory_robotic_fetch.py` and `aria_live_streamer.py`